integr8ly/manifests
# INTLY-2757 bump integreatly minor version for user-sso

## What
bump the version of integreatly
- [x] Update the makefile 
- add a new csv for integreatly

## Verification Steps
1. Push the changes above to your app registry
 - Run `make push/all`
 - **Verify**: Integreatly Operator (1.8.0) should be available in your app registry
2. Create an `OperatorSource` with your app registry
3. Install the operator to a specific namespace
**Verify** user-sso exists and the routes are working
